package.json

```diff
@@ -56,7 +56,7 @@
     "jade": "^1.11.0",
     "jade-loader": "~0.8.0",
     "jest": "^23.4.1",
-    "jest-silent-reporter": "^0.0.5",
+    "jest-silent-reporter": "^0.1.1",
```

Comment:
Notice that we forgot to remove this dependency

So if we don't want to use this reporter, let me know and i'll update the PR to just remove the dep